gxformat2/model.py

```diff
@@ -44,11 +57,70 @@ def ensure_step_position(step: dict, order_index: int):
         }
 
 
-def inputs_as_steps(workflow_dict: dict):
+def native_input_to_format2_type(step: dict, tool_state: dict) -> str:
+    """Return a Format2 input type ('type') from a native input step dictionary."""
+    module_type = step.get("type")
+    if module_type == 'data_collection_input':
+        format2_type = 'collection'
+    elif module_type == 'data_input':
+        format2_type = 'data'
+    elif module_type == "parameter_input":
+        native_type = cast(str, tool_state.get("parameter_type"))
+        format2_type = native_type
+        if native_type == "integer":
+            format2_type = "int"
+        format2_type = format2_type
```

Comment:
This looks strange. The above if should also moved up one line maybe?

Reply:
Good catch - I'll fix this.